example/example-Tzip16OffChainTwo.ts

```diff
@@ -0,0 +1,63 @@
+import { MichelsonMap, TezosToolkit } from '@taquito/taquito';
```

Comment:
I think the title should be onChain instead of OffChain as the metadata is stored on-chain.